.circleci/config.yml

```diff
@@ -86,6 +86,14 @@ commands:
             echo "Running maven-java with SDK version: '$SDK_VERSION'"
             mvn versions:set -DnewVersion=$SDK_VERSION
             mvn verify install
+
+  build-sbt-scala:
+    description: Build sbt plugin and scala artifacts
+    steps:
+      - run:
+          name: publish local
+          command: sbt publishLocal
```

Comment:
I think it might be better to `publishLocal` before above maven publish, because `mvn version:set` modifies some files and the version number will not become the same here.

Reply:
Ah, that's probably causing the problem I mentioned that I didn't quite understand yesterday. thx